src/main/java/com/labs/introtoprogramming/lab4/image/Pixel.java

```diff
@@ -30,6 +30,29 @@ public byte blue() {
     return blue;
   }
 
+  public Pixel brighter(double factor) {
+    int i = (int)(1.0/(1.0-factor));
```

Comment:
Maybe rename to something like intensity to better understand what is doing

Reply:
Done.